packages/react-ui/src/app/builder/pieces-selector/no-results-found.tsx

```diff
@@ -0,0 +1,81 @@
+import { t } from 'i18next';
+import { SearchX, WandSparkles } from 'lucide-react';
+
+import { useEmbedding } from '@/components/embed-provider';
+import { Button } from '@/components/ui/button';
+import { flagsHooks } from '@/hooks/flags-hooks';
+import { platformHooks } from '@/hooks/platform-hooks';
+import { PieceSelectorOperation } from '@/lib/types';
+import { ApFlagId, FlowOperationType, supportUrl } from '@activepieces/shared';
+
+import { useBuilderStateContext } from '../builder-hooks';
+
+import { AskAiButton } from './ask-ai';
+
+const RequestPieceButton = () => {
```

Comment:
I think you can inline this and just extract onClick function to above, openFeatureRequests()